### DISCUSSION TITLE
[Feature] SmilingWolf ML Model

### DISCUSSION THREAD

**Discussion by ThesiiNCey** [immich-app/immich#12105] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hey my lovely Immich Developers

is it possible that you add this model to your ML repo for Immich ?

https://huggingface.co/SmilingWolf/wd-swinv2-tagger-v3

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mertalev** [comment]
This is an image tagging model, so not suitable for similarity search. It has to have aligned textual and visual components that both output embeddings.

Also, we can only consider general-purpose models. Domain-specific models are out of scope.

If you really want to, though, it's technically possible to replace a model in the model cache with a different one as long as it behaves like the other model (same dimension size; same input/output shape, type and name; same config layout). This is only if you're very familiar with wrangling models.
